admin/base/settings/defaults.py

```diff
@@ -31,10 +31,14 @@
     'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
+    'django.contrib.messages',
+    'django.contrib.sessions',
     'django.contrib.staticfiles',
 
     # 3rd party
+    'admin.common_auth',
```

Comment:
@Ghalko I needed it there for migrating, since we have AdminUser defined in models.py in common_auth.
However, as you mentioned about just using Django's built in User, we could scratch that and wouldn't have to include common_auth in installed_apps. 
The original idea behind extending the User model was to add additional fields if it were needed either now (for example, the now-deprecated idea of storing osf-user ids) or later.